__tests__/app.test.js

```diff
@@ -100,4 +100,38 @@ describe("GET /api/articles/article_id", () => {
       expect(msg).toBe('bad request')
     })
   })
+})
+
+describe("GET /api/articles", () => {
+  test.only('200: returns an array of all articles sorted by date', () => {
+    return request(app)
+    .get("/api/articles?sort_by=created_at")
+    .expect(200)
+    .then(({body}) => {
+        const { articles } = body;
+        expect(articles).toHaveLength(13);
+        expect(articles).toBeSortedBy('created_at')
+        articles.forEach((obj) => {
+            expect(obj).toMatchObject({
+                author: expect.any(String),
+                title: expect.any(String),
+                article_id: expect.any(Number),
+                topic: expect.any(String),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+                article_img_url: expect.any(String),
+                comment_count: expect.any(String)
+        })
+      })
+    })
+  })
+  test('400: returns a 400 error if there is a bad request in path', () => {
```

Comment:
At this point this test isn't really needed as we aren't on queries for this endpoint. However, you are correct you will need a 400 eventually.